src/standalone/Vegetation/Canopy.jl

```diff
@@ -692,10 +693,37 @@ function CanopyModel{FT}(;
         @assert typeof(photosynthesis) <: PModel{FT} "When using PModelConductance for stomatal conductance, you must also use PModel for photosynthesis"
     end
 
+    # Height consistency check between biomass and hydraulics:
+    #
+    # biomass.height:
+    #   - Used for: Aerodynamic calculations (displacement height, roughness length)
```

Comment:
I see that the biomass.height is used in `displacement_height`, but it looks like the roughness lengths still independently come from the toml dict [here](https://github.com/CliMA/ClimaLand.jl/blob/7787892f6787c32f326cf06e5011c5ef69772dea/src/standalone/Vegetation/Canopy.jl#L799-L800)

Reply:
Yes, I think this PR just adjusts displacement height. The roughness lengths are either constant (set in the toml, the default) or computed  like so: https://github.com/CliMA/ClimaLand.jl/blob/7787892f6787c32f326cf06e5011c5ef69772dea/experiments/long_runs/snowy_land_pmodel.jl#L97-L99

but they are required to be scalar.

I am adding a PR which allows these to be spatially varying, among other things.